packages/compiler/src/jit_compiler_facade.ts

```diff
@@ -337,6 +341,76 @@ function convertOpaqueValuesToExpressions(obj: {[key: string]: OpaqueValue}):
   return result;
 }
 
+function convertDeclareComponentFacadeToMetadata(
+    declaration: R3DeclareComponentFacade, typeSourceSpan: ParseSourceSpan,
+    sourceMapUrl: string): R3ComponentMetadata {
+  const {template, interpolation} = parseJitTemplate(
+      declaration.template.source, declaration.type.name, sourceMapUrl,
+      declaration.preserveWhitespaces ?? false, declaration.interpolation);
+
+  return {
+    ...convertDeclareDirectiveFacadeToMetadata(declaration, typeSourceSpan),
+    template,
+    styles: declaration.styles ?? [],
+    directives: (declaration.directives ?? []).map(convertUsedDirectiveDeclarationToMetadata),
+    pipes: convertUsedPipesToMetadata(declaration.pipes),
+    viewProviders: declaration.viewProviders !== undefined ?
+        new WrappedNodeExpr(declaration.viewProviders) :
+        null,
+    animations: declaration.animations !== undefined ? new WrappedNodeExpr(declaration.animations) :
+                                                       null,
+    changeDetection: declaration.changeDetection !== undefined ? declaration.changeDetection :
+                                                                 ChangeDetectionStrategy.Default,
```

Comment:
I think this can just be:

```suggestion
    changeDetection: declaration.changeDetection ?? ChangeDetectionStrategy.Default,
```

Reply:
A yes that is appropriate here, changed.